scripts/branch-name-lint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+branch_name=$(git rev-parse --abbrev-ref HEAD)
+
+
+regex="^(feat|bugfix|hotfix|chore|test)/(issue-[0-9]+|no-ref)/[a-z0-9\-]+$"
```

Comment:
Do you want these types to match the commit or maybe we could leave off the type for the branch name?